_includes/help/nav_sidenav.html

```diff
@@ -15,9 +14,9 @@
       {% else %}
         {% assign page_lang = '' %}
       {% endif %}
-      <a href="{{ subpage_slug | append: '/' | prepend: '/help/' | locale_url }}"
-        class="{% if page.url contains subpage_slug %}usa-current{% endif %}">
-        {{ site.data[page.lang]["settings"]["help_subpages"][subpage] }}
+      <a href="{{ subpage.url | prepend: '/help/' | locale_url }}"
```

Comment:
Arguably we could put the `/help/` into the `url` so we don't need to `prepend` it here? Especially if the goal is to avoid constructing a URL from the slug.

Reply:
ah, good call, added `help` to the URL so we can remove the prepend in [eb64241](https://github.com/GSA-TTS/identity-site/pull/1236/commits/eb642419eddeee667ea3db756b4996a43b0a8504)